components/trainee/About_trainee.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import { useColorScheme } from 'react-native';
+
+interface Profile {
+  name?: string;
+  user?: {
+    email?: string;
+  };
+  phoneNumber?: string;
+  address?: string;
+  githubUsername?: string;
+  resume?: string;
+  cover?: string;
+  biography?: string;
+}
+
+interface AboutTraineeProps {
+  profile: Profile;
+  bgColor: string;
+  textColor: string;
+}
+
+const AboutTrainee: React.FC<AboutTraineeProps> = ({ profile, bgColor, textColor }) => {
+  const colorScheme = useColorScheme();
+
+  return (
+    <>
+      <View className={`${bgColor} w-[100%] p-6 rounded-md`}>
+        <Text className={`${textColor} text-xl`}>BASIC INFORMATION</Text>
+        <View className="flex-row gap-5 m-2">
+          <Ionicons name="person" size={20} color={colorScheme === 'dark' ? 'white' : 'black'} />
+          <View className="flex-row gap-2">
+            <Text className={`${textColor} text-lg`}>
+              {profile.name ? profile.name : 'Unavailable'}
+            </Text>
+          </View>
+        </View>
+        <View className="flex-row gap-5 m-2">
+          <Ionicons name="mail" size={20} color={colorScheme === 'dark' ? 'white' : 'black'} />
+          <Text className={`${textColor} text-lg`}>
+            {profile.user?.email ? profile.user?.email : 'Unavailable'}
+          </Text>
+        </View>
+        <View className="flex-row gap-5 m-2">
+          <Ionicons name="call" size={20} color={colorScheme === 'dark' ? 'white' : 'black'} />
+          <Text className={`${textColor} text-lg`}>
+            {profile.phoneNumber ? profile.phoneNumber : 'Unavailable'}
+          </Text>
+        </View>
+        <View className="flex-row gap-5 m-2">
+          <Ionicons name="home" size={20} color={colorScheme === 'dark' ? 'white' : 'black'} />
+          <Text className={`${textColor} text-lg`}>
+            {profile.address ? profile.address : 'Unavailable'}
+          </Text>
+        </View>
+        <View className="flex-row gap-5 m-2">
+          <Ionicons name="logo-github" size={20} color={colorScheme === 'dark' ? 'white' : 'black'} />
+          <Text className={`${textColor} text-lg`}>
+            {profile.githubUsername ? profile.githubUsername : 'Unavailable'}
+          </Text>
+        </View>
+        <Text className={`${textColor} text-xl`}>RESUME</Text>
+        <Text className={`${textColor} text-md m-2`}>
+          {profile.resume ? profile.resume : 'No Resume uploaded yet'}
+        </Text>
+      </View>
+      <View className={`${bgColor} w-[100%] p-6 rounded-md mt-5`}>
```

Comment:
here add same margin buttom like mb-14 , because on some devices like mine i can't scrolling down to the end:

![image](https://github.com/user-attachments/assets/18f3f248-d882-425e-b60c-8aacc99229c5)

but if you add mb-14 it look like this:

![image](https://github.com/user-attachments/assets/054eeed7-9f05-411f-96bc-1694e070d867)

Now i can see the page is ended

Reply:
Done!